GVFS/GVFS/CommandLine/DiagnoseVerb.cs

```diff
@@ -88,15 +88,18 @@ protected override void Execute(GVFSEnlistment enlistment)
                     () =>
                     {
                         // .gvfs
-                        this.CopyAllFiles(enlistment.EnlistmentRoot, archiveFolderPath, GVFSConstants.DotGVFS.Root, copySubFolders: false);
-
-                        // filter
-                        this.FlushFilterLogBuffers();
-                        string system32LogFilesPath = Environment.ExpandEnvironmentVariables(System32LogFilesRoot);
-
-                        // This copy sometimes fails because the OS has an exclusive lock on the etl files. The error is not actionable
-                        // for the user so we don't write the error message to stdout, just to our own log file.
-                        this.CopyAllFiles(system32LogFilesPath, archiveFolderPath, GVFSPlatform.Instance.KernelDriver.DriverLogFolderName, copySubFolders: false, hideErrorsFromStdout: true);
+                        this.CopyAllFiles(enlistment.EnlistmentRoot, archiveFolderPath, GVFSConstants.DotGVFS.Root, copySubFolders: false);
+
+                        if (!GVFSPlatform.Instance.IsUnderConstruction)
```

Comment:
I'm not sure we need an under construction flag here - we could just check if the log folder exists.

Reply:
> I'm not sure we need an under construction flag here - we could just check if the log folder exists.

We don't know the log path to return on Mac (yet), but I could switch from using a `LogsFolderPath` property to a `TryGetLogsFolderPath` method and eliminate the under construction flag (I'm not sure that's actually buying us much though).  

Thoughts?

Reply:
Or just return null for now? And this will get cleaned up in the very next PR right? So you could also just leave it as is.

Reply:
> Or just return null for now? And this will get cleaned up in the very next PR right? So you could also just leave it as is.

I'll go with whatever is cleanest.  I was hoping to avoid code that looks permanent (e.g. checking if the path is `null`) that was added *only* to avoid calling `IsUnderConstruction` (I'd rather be more explicit about why we have a check and use `IsUnderConstruction`).

As you mentioned, this will get cleaned up as soon as we have kext logging (which should be soon) and so I don't think it's a big deal to leave in the `IsUnderConstruction` (assuming that ends up being the cleanest option).